Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/test_data/activity_dns_data.json

```diff
@@ -0,0 +1,50 @@
+{
+    "meta": {},
+    "data": [
+        {
+            "returncode": 0,
+            "externalip": "1.1.1.1",
+            "allapplications": [],
+            "date": "2022-09-12",
+            "internalip": "10.10.10.162",
```

Comment:
please replace the 'internalip' with a dummy IP - such as 1.1.1.1

Reply:
@ShacharKidor modified